src/types/dynamoDBParams.ts

```diff
@@ -0,0 +1,5 @@
+import { DataRequestParams } from './dataRequestParams'
+
+export interface DynamoDBParams extends DataRequestParams {
```

Comment:
Does the table we use have a more descriptive name? `DynamoDBParams` feels a bit generic to me and could be anything. Especially considering we're introducing another table soon for the secure fraud site stuff.

Reply:
As above, we won't need this interface at all once we stop deserializing stuff

Reply:
I've kept the interface but used it in the dynamoDBGet service - this will be used elsewhere so  needed to be expanded to account for other fields such as the athenaQueryId